tests/election.py

```diff
@@ -64,11 +60,14 @@ def run(args):
         for _ in range(nodes_to_stop):
             # Note that for the first iteration, the primary is known in advance anyway
             LOG.debug("Find freshly elected primary")
-            primary, current_term = network.find_primary()
+            # After a view change in pbft, finding the new primary takes longer
```

Comment:
@ashamis @olgavrou Do we know why that is?

Reply:
what triggers a view change in PBFT is when backups think they should be sent a pre-prepare with a specific request but they do not receive it within a given time. So the sleep that helps in the Raft case is not useful for PBFT.

Reply:
In pbft "read" requests make a full pbft round, not similar to raft where they are actual read requests. If we have recently view changed then replicas need to execute requests that they had to roll back before they changed the view, so it might take some time before new requests can be executed.